client/app/scripts/superdesk-search/views/item-search.html

```diff
@@ -72,14 +72,20 @@
                 <input type="text" id="search-storytext" ng-model="meta.body_html">
             </div>
         </div>
-
+        <div class="field">
+            <label for="search-byline">
+                {{:: 'Byline' | translate}}
+            </label>
+            <input type="text" id="search-byline" ng-model="meta.byline">
+        </div>
         <div class="field" ng-if="repo.archive">
             <label for="search-creator">
                 {{:: 'Creator' | translate}}
             </label>
-            <input type="text" id="search-creator" ng-model="meta.original_creator">
+            <select id="search-creator" ng-model="meta.original_creator" ng-options="user._id  as user.display_name for user in userList track by user._id">
```

Comment:
When using ng-options, using both the `select as` and `track by` parts in the expression might result in unexpected behavior as explained in the [ng-options documentation]( https://docs.angularjs.org/api/ng/directive/ngOptions). It's thus probably best to remove the `track by` part in order to avoid potentially introducing a bug if the expression gets changed in the future.